src/tests/SharpRaven.UnitTests/SchemaHelper.cs

```diff
@@ -72,11 +72,18 @@ public static string GetSchemaPath()
             var directory = new DirectoryInfo(Environment.CurrentDirectory);
             FileInfo file = null;
 
+#if (net35)
+            while (directory != null && directory.Exists &&
+                   (file = directory.GetFiles("*.json")
+                       .FirstOrDefault(f => f.FullName.EndsWith("schema.json"))) == null)
+                            directory = directory.Parent;
+#else
```

Comment:
Nah, I'm not fuzzy about it. We've already disabled C# 6 for the entire solution, so I don't really care how "modern" the code is as long as it works.